library/lcd/lcd_comm_rev_a.py

```diff
@@ -41,6 +41,8 @@ def __init__(self, com_port: str = "AUTO", display_width: int = 320, display_hei
                  update_queue: queue.Queue = None):
         LcdComm.__init__(self, com_port, display_width, display_height, update_queue)
         self.openSerial()
+        self.idVendor = 0x1a86
+        self.idProduct = 0x5722
```

Comment:
The idVendor and idProduct may vary, I have seen at least 2 different values for Turing smart screens. It would be better to read them directly from the device after opening serial (can be done in `openSerial()` for all devices)

Reply:
Ok, ! Good point, there is a variable holding this values in serial ? In don't know.

Reply:
Unfortunately not in Serial, but it can be retrieved from COM ports list:
```python
from serial.tools.list_ports import comports

[...]

        for com_port in comports():
            if com_port.device == self.com_port:
                self.idVendor = com_port.vid
                self.idProduct = com_port.pid
                break
```
It can be added in lcd_comm.openSerial() after COM port discovery

Reply:
Great ! I will replace that part.